src/cmd/cmdgazebo.rb.in

```diff
@@ -446,13 +446,6 @@ has properly set the DYLD_LIBRARY_PATH environment variables."
       # Neither the -s nor -g options were used, so run both the server
       # and gui.
       if options['server'] == 0 && options['gui'] == 0
-
-        if plugin.end_with? ".dylib"
-          puts "`ign gazebo` currently only works with the -s argument on macOS.
```

Comment:
maybe we should add here a message about how we should launch `ign gazebo` in MacOS (server and gui in different terminals)

Reply:
Good suggestion, I'll add that back in. The reason btw is because macOS does not support multi-threading `fork - exec`, at least it's not safe, and versions since Catalina have prevented its use. The same issue crops up in Python; the solution there is to use multi-processing. I am not familiar enough with ruby to know its equivalent.

Reply:
Updated. Message reads:

```bash
$ ign gazebo
On macOS `ign gazebo` currently only works with either the -s argument
or the -g argument, you cannot run both server and gui in one terminal. 
See https://github.com/ignitionrobotics/ign-gazebo/issues/44 for more info.
```